RELEASE_NOTES

```diff
@@ -1,6 +1,36 @@
 This file contains a description of the major changes to the EESSI test suite.
 For more detailed information, please see the git log.
 
+v0.4.0 (26 September 2024)
+---------------------
+
+This is a minor release of the EESSI test-suite
+
+New tests:
+
+* PyTorch (with torchvision) (#130)
+* LAMMPS (#131, #169)
+* CP2K (#133)
+
+Bug fixes:
+* Find and report duplicate modules (#167)
+* Specify memory as an extra resource in all ReFrame configs (#180)
+* Make req_memory_per_node hook print a warning if memory resource is not configured in ReFrame config (#182)
+* No longer test import of eessi.testsuite.utils directly in CI, as it fails (by design) for the ReFrame 4.6.2 version deployed in EESSI (#183)
+* Update scorecards action to fix scorecards failure (#184)
```

Comment:
This is internal business, I wouldn't mention that in the release notes

Reply:
Removed